### PR TITLE
[Test] Add link to Raisinbread modification instructions to the CONTRIBUTING doc

### PR DESCRIPTION
**Pull request by johnsaigle**
## Brief summary of changes

Update our CONTRIBUTING doc to include a directive on updating Raisinbread when SQL changes are made.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ issues and/or Redmine tickets (if applicable).
     `SQL/New_patches/` directory.
     4. For SQL patches that are _optional_ (e.g. those that perform some 
             cleanup), place them in `SQL/Cleanup_patches/`.
+    5. The files for our test database ("Raisinbread") must be updated.
```
Comment (cmadjar) [inline CONTRIBUTING.md]: Actually, maybe modify the sentence a bit to say:
"The files for our test database (Raisinbread) must be updated if they were modified. Detailed instructions..."

Comment (johnsaigle) [reply CONTRIBUTING.md]: If you look that's what line 78 says. This is a list under that.

Comment (cmadjar) [reply CONTRIBUTING.md]: actually, line 78 mention only database table changes, not changes done to the actual files.

**Comment by johnsaigle** [comment]
@ridz1208 Can you merge this?

**Pull request merged**